coops-ui/src/resources/business-config.ts

```diff
@@ -60,7 +69,32 @@ export const configJson = [{
       displayName: 'Change Of Directors',
       certifyText: 'Note: It is an offence to make a false or misleading statement in ' +
            'respect of a material fact in a record submitted to the Corporate Registry for filing. ' +
-           'See section 78 of the Cooperative Association Act.'
+           'See section 78 of the Cooperative Association Act.',
```

Comment:
Looks like "section" is capitalized almost everywhere else. Please fix globally.